appeals/web/src/server/appeals/appeals.types.d.ts

```diff
@@ -9,6 +9,12 @@ export interface CheckboxItemParameter {
 	checked?: boolean;
 }
 
+export interface SelectItemParameter {
```

Comment:
Just a suggestion, this might be better off in `lib/mappers/mapper-types-jsdoc.js`